src/pages/ResourceView.tsx

```diff
@@ -0,0 +1,31 @@
+import { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { useLoaderData, useNavigate } from 'react-router-dom'
+import ResourceCard from 'src/components/Resources/ResourceCard'
+import { ResourceLoaderReturn } from 'src/loaders/resourceLoader'
+import Resource from 'src/types/Resource'
+
+export default function ResourceView() {
+  const navigate = useNavigate()
+  const { t } = useTranslation()
+  const data = useLoaderData() as ResourceLoaderReturn
+  const [resource, setResource] = useState(data.resource)
+  const onUpdateResource = (updatedResourceData: { resource: Resource }) => {
+    const updatedResource = updatedResourceData.resource
+    const newResource = { ...resource, ...updatedResource }
+
+    setResource(newResource)
+  }
+
+  return (
+    <div>
+      <a role="button" onClick={() => navigate(-1)}>
+        <i className="bi bi-chevron-left align-middle" />
+        {t('ui.back')}
+      </a>
+      <div className="vertical-rhythm">
```

Comment:
I think if you remove this div and add the `vertical-rhythm` className to the top-level div, it'll sort out the spacing issue.

Reply:
I've done that but it still seems to not add any spacing between the back button and the resource card.